content/concepts/payment-system-basics/ledgers.md

```diff
@@ -31,6 +31,6 @@ All objects in a ledger' state tree have a unique ID. This field is returned as
 ![Diagram: rippled uses SHA-512Half to generate IDs for ledger objects. The space key prevents IDs for different object types from colliding.](img/ledger-indexes.png)
```

Comment:
Pitch: the "Object IDs" section should be its own page in the Ledger Reference section, because it defines the generic form of all the IDs that are defined in more detail on the "Ledger Object Types" pages. 

In other words, it's highly relevant within those pages and not very useful in the "Concepts" area.

Reply:
Done. Does the location of the Ledger Object IDs page in the hierarchy work for you?